src/client/java/teammates/client/scripts/sql/VerifySectionAttributes.java

```diff
@@ -0,0 +1,56 @@
+package teammates.client.scripts.sql;
+
+import java.util.stream.Stream;
+
+import teammates.storage.entity.Course;
+import teammates.storage.entity.CourseStudent;
+
+/**
+ * Class for verifying section attributes.
+ */
+@SuppressWarnings("PMD")
+public class VerifySectionAttributes
+        extends VerifyNonCourseEntityAttributesBaseScript<Course, teammates.storage.sqlentity.Section> {
+
+    public VerifySectionAttributes() {
+        super(Course.class,
```

Comment:
Actually I realized the fix is quite fast. Could we do a quick fix and query the SQL course instead, then make sure sections are equal

Reply:
Done!